sonarapp/Program.cs

```diff
@@ -10,6 +10,19 @@ class Program
     {
         static void Main(string[] args)
         {
+            string test = "This is a test";
+
+            Boolean what = (new verify( )).verifywhat("whatif");
         }
     }
+
+    public class verify
+    {
+        string name;
+
+        public Boolean verifywhat(string name)
+        {
+            return this.name == name ? true:false;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the unnecessary Boolean literal(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=csharpsquid%3AS1125)